tools/validate_spec.py

```diff
@@ -180,139 +224,41 @@ def __exit__(
             self.file.write("\n]")
             self.file.close()
 
-    def write(self, validation_result: "ValidationResult") -> None:
+        if exception_value is None and self.classified_incorrectly != 0:
+            raise ValidationError(
+                f"{self.classified_incorrectly} messages were classified incorrectly"
+            )
+
+    def write_result(self, validation_result: ValidationResult) -> None:
+        self.__write_console_output(validation_result)
+        if self.file is not None:
+            self.__write_json_output(validation_result)
+
+    def __write_console_output(self, result: ValidationResult) -> None:
+        if result.validation_success:
+            print(f"{str(result.message_file_path):<80} {'PASSED'}")
```

Comment:
Putting a string literal into curly brackets is useless.

Reply:
Fixed in e701f54